teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name)
+    super(age, name)
```

Comment:
- [ ] Kindly also inherit the `parent_permission` parameter from the `Person` class. Kindly add it to `super`

Reply:
Thanks! I don't think that this is necessary. a parent would have control of his child (student), and may not directly control the teacher.

Reply:
- [ ] Hello @stephnna  the inheritance of the `parent_permission` it's not to give the teacher permission but for the student in the teacher's classroom, and that's why the `age` plays a role `if age <= 18`.

Reply:
Hi @dasileker 
Wow, your explanation is clear, thanks! Done!